### DISCUSSION TITLE
Inaccurate comment on Armament::Fire

### DISCUSSION THREAD

**Discussion by warp-core** [endless-sky/endless-sky#6729] [General]
This doesn't seem important enough for an issue, I guess this is what discussions are for?

https://github.com/endless-sky/endless-sky/blob/29dce86e88a608937c47ba9a571db2588978c460/source/Armament.cpp#L209-L211

The comment says it returns false if the weapon did not fire if it was not ready, but this is not the case, the method has no return value and always exits the same way.
The following method, Armament::FireAntiMissile does return false if it is not ready.
Is the comment intended to refer to that behaviour and not the behaviour of Armament::Fire? Should Armament::Fire replicate that behaviour or is the current behaviour preferable and the comment is what needs changing?

The same contradiction is present in the comment and signature for this method in Armament.h:
https://github.com/endless-sky/endless-sky/blob/29dce86e88a608937c47ba9a571db2588978c460/source/Armament.h#L67-L69

As far as I can tell, this has been the case since MZ originally created the Armament class.

**Comment by TomGoodIdea** [comment]
fixed in #10692 ~~didn't even know there was a discussion lol~~
